ontology/yaml/resources/SAFETY/entity_types/ABSTRACT.yaml

```diff
@@ -79,18 +79,56 @@ FA2X:
   uses:
   - fire_alarm_1
   - fire_alarm_2
+  opt_uses:
+  - primary_fire_alarm
+  - secondary_fire_alarm
     
 WLS:
   id: "944988050314756096"
   description: "Water leak location monitoring."
   is_abstract: true
   uses:
   - water_leak_cable_distance_sensor
-  
+
 LM:
   id: "15773300929664647168"
   description: "Tank level monitoring."
   is_abstract: true
   uses:
   - level_status 
 
+HDS:
```

Comment:
These types are GENERALTYPEs, so put them in that file.

Reply:
Noted sir, will do